src/sentry/static/sentry/app/views/organizationIntegrations/constants.tsx

```diff
@@ -7,3 +7,45 @@ export const colors = {
   [NOT_INSTALLED]: 'gray2',
   [PENDING]: 'yellowOrange',
 };
+
+/**
+ * Integrations in the integration directory should be sorted by their popularity (weight).
+ * These weights should be hardcoded in the application itself.
+ * We can store this in a map where the key is the integration slug and the value is an integer and represents the weight.
+ * The weights should reflect the relative popularity of each integration.
+ */
+
+export const popularityWeights = {
+  slack: 25606,
+  github: 15161,
+  webhooks: 15131,
+  jira: 12875,
+  bitbucket: 11070,
+  trello: 10938,
+  heroku: 10904,
+  gitlab: 10731,
+  twilio: 9149,
+  pivotal: 8734,
```

Comment:
@NisanthanNanthakumar How did you come up with these numbers? Questioning why some of these plugins are so high. The popularity shouldn't just be based on the total number of installs (which favors older integrations) but how many are being installed at the present moment. Also, we should remove the soon to removed plugins:

- campfire
- irc
- youtrack
- flowdock

Reply:
@scefali I got it from @jonesphillip

Reply:
@scefali I'll remove those plugins

Reply:
@NisanthanNanthakumar ok but where did these numbers come from? @jonesphillip

Reply:
@scefali it came from looker. I'll slack you the details